ulimit_linux_test.go

```diff
@@ -0,0 +1,17 @@
+// +build linux
+
+package units
+
+import (
+	"testing"
+)
+
+func TestParseUlimitTooBig(t *testing.T) {
+	u, err := ParseUlimit("nofile=512:1000024")
```

Comment:
I noticed CI isn't running on this repo (have to check why), but testing this locally in a container, the test failed.


```
--- FAIL: TestParseUlimitTooBig (0.00s)
    ulimit_linux_test.go:15: expected invalid hard limit
```

I added a debug line to `Verify()` (see below), which explains why it failed, because on this machine the limit is higher than what the test expects;

```
ulimit hard limit (1000024) is less than 1048576
```

Perhaps this test needs to dynamically set the value to make sure it's higher than the max. That's a bit ugly, so other suggestions welcome 😅)


```patch
diff --git a/ulimit_linux.go b/ulimit_linux.go
index a56f361..36a4b28 100644
--- a/ulimit_linux.go
+++ b/ulimit_linux.go
@@ -13,6 +13,7 @@ func (u *Ulimit) Verify() error {
 		if u.Hard > int64(rlimit.Max) {
 			return fmt.Errorf("ulimit hard limit (%d) must be less than or equal to hard limit for the current process %d",
 u.Hard, rlimit.Max)
 		}
+		fmt.Printf("ulimit hard limit (%d) is less than %d\n", u.Hard, rlimit.Max)
 	}
 	return nil
 }
```

Reply:
@thaJeztah seems this issue was addressed -- the test case now gets the max value from the kernel.